source/Optimization.rst

```diff
@@ -0,0 +1,37 @@
+Optimisation
+===============
+
+
+PerTickCache
```

Comment:
First please describe what the per-tick cache actually _is_. Go through the ``documentation.md`` file, in the "Help and FAQs" section there's a brief description of the per-tick cache that you can start off by describing. Secondly, why are you capitalising it like this? You aren't really describing a computational _function_ here, more a feature.